lib/open_food_network/variant_stock.rb

```diff
@@ -77,7 +84,7 @@ def count_on_hand=(new_level)
     # track_inventory_levels only. It was initially introduced in
     # https://github.com/openfoodfoundation/spree/commit/20b5ad9835dca7f41a40ad16c7b45f987eea6dcc
     def on_demand
-      warn_deprecation(__method__)
+      warn_deprecation(__method__, 'Spree::Config[:track_inventory_levels]')
```

Comment:
This is not right. Track inventory levels is a global setting that we don't use at all. Alternatives are `in_stock?` and `stock_items.first.backorderable?`.

Reply:
> Track inventory levels is a global setting that we don't use at all

is that explained/documented somewhere?